core/src/main/scala/Elo.scala

```diff
@@ -46,3 +53,50 @@ object Elo extends OpaqueInt[Elo]:
           case Outcome.Points.Half => 0
           case Outcome.Points.One  => 1
       (ratings + points * winBonus) / games.size
+
+  // 8.1.2 FIDE table
+  val conversionTableFIDE: List[(Range, Double)] = List(
```

Comment:
could we make this a `Map`, so the look up can be a faster?

Reply:
There are no lookups, only traversing, as the "keys" are ranges:

```scala
conversionTableFIDE.collectFirst {
  case (range, pd) if range.contains(absRatingDiff) => pd
}
```